modules/storage-persistent-storage-pv.adoc

```diff
@@ -249,12 +256,14 @@ cluster.
 
 |===
 
-You can view the name of the PVC that is bound to the PV by running the following command:
+=== Last phase transition time
+The `LastPhaseTransitionTime` field has a timestamp that updates every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`). To find the time of the last phase transition for a PV, run the following command:
 
 [source,terminal]
 ----
-$ oc get pv <pv-claim>
+$ oc get pv <pv-name. -o json | jq '.status.lastPhaseTransitionTime' <1>
```

Comment:
@lpettyjo little typo here 
<pv-name. instead of <pv-name>

Reply:
Good catch. Fixed.